src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,150 @@
+
+    ```java
+    dependencies {
+        implementation 'io.opentelemetry:opentelemetry-api'
+        implementation 'io.opentelemetry:opentelemetry-sdk'
+    }
+    ```
+    
+An example completed file might look something like this:
+
+    ```java
+    plugins {
+        id 'java-library'
+        id 'org.springframework.boot'
+    }
+
+    bootRun {
+        mainClass.set 'io.opentelemetry.example.micrometer.Application'
+    }
+
+    dependencies {
+        //OpenTelemetry API and SDK
+        implementation 'io.opentelemetry:opentelemetry-api'
+        implementation 'io.opentelemetry:opentelemetry-sdk'
+        //Alpha modules where Micrometer instrumentation appears
+        implementation 'io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5'
+        implementation 'io.opentelemetry:opentelemetry-exporter-otlp'
+
+        implementation platform(SpringBootPlugin.BOM_COORDINATES)
+        implementation 'org.springframework.boot:spring-boot-starter-web'
+        implementation 'org.springframework.boot:spring-boot-starter-actuator'
+        implementation 'org.springframework.boot:spring-boot-starter-aop'
+    }
+    ```
+
+    </Step>
+    <Step>
+
+### Configure Micrometer to forward data to New Relic
+
+Below is an example code snippet that directs Micrometer to use the OpenTelemetry Micrometer bridge. This snippet updates your code so OpenTelemetry can:
+
+* Detect Micrometer data
+* Forward that data to New Relic
+
+```java
+public OpenTelemetry openTelemetry() {
+    return OpenTelemetrySdk.builder()
+        .setMeterProvider(
+            SdkMeterProvider.builder()
+                .setResource(
+                    Resource.getDefault().toBuilder()
+                        .put("service.name", "micrometer-shim")
+                        // Include instrumentation.provider=micrometer to enable micrometer metrics
+                        // experience in New Relic
+                        .put("instrumentation.provider", "micrometer")
+                        .build())
+                .registerMetricReader(
+                    PeriodicMetricReader.builder(
+                            OtlpHttpMetricExporter.builder()
+                                .setEndpoint("https://otlp.nr-data.net")
+                                .addHeader(
+                                    "api-key",
+                                    Optional.ofNullable(System.getenv("NEW_RELIC_LICENSE_KEY"))
+                                        .filter(str -> !str.isEmpty() && !str.isBlank())
+                                        .orElseThrow())
+                                // IMPORTANT: New Relic requires metrics to be delta temporality
+                                .setAggregationTemporalitySelector(
+                                    AggregationTemporalitySelector.deltaPreferred())
+                                // Use exponential histogram aggregation for histogram instruments
+                                // to
```

Comment:
Same as above: what does it mean to 'use histogram aggregation', can we unspool this so the comments are a bit more helpful? (i see that this is the name itself, but i think the comments can elaborate on the name)

Reply:
When micrometer records measurements against its timer instrument type, e.g. for `http.server.requests`, those are aggregated and exported as a [histogram with explicit bucket boundaries](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#explicit-bucket-histogram-aggregation) by default. Here we're instructing the SDK to use [exponential bucket histogram](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#base2-exponential-bucket-histogram-aggregation) instead, which produces a much more detailed / useful distribution which automatically scales to record the range of measurements recorded to the instruments.

There's obviously a lot to unpack with that, which to do properly would take a few paragraphs. Not sure its worth it to elaborate.